{"name": "Получение заказов авторизованного пользователя", "status": "failed", "statusDetails": {"message": "AssertionError: assert 403 == 200\n +  where 403 = <Response [403]>.status_code", "trace": "self = <Diplom_2.tests.test_get_user_orders.TestGetUserOrders object at 0x00000203BBC12210>\n\n    @allure.title('Получение заказов авторизованного пользователя')\n    def test_get_orders_authorized(self):\n        response = self.api.get_user_orders(self.token)\n>       assert response.status_code == 200\nE       assert 403 == 200\nE        +  where 403 = <Response [403]>.status_code\n\ntests\\test_get_user_orders.py:23: AssertionError"}, "start": 1751988283362, "stop": 1751988283572, "uuid": "990dd681-affc-4adf-9728-650eefe6880a", "historyId": "c61394d2ac4d3f409a6b251ffd517580", "testCaseId": "c61394d2ac4d3f409a6b251ffd517580", "fullName": "tests.test_get_user_orders.TestGetUserOrders#test_get_orders_authorized", "labels": [{"name": "feature", "value": "Получение заказов пользователя"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_get_user_orders"}, {"name": "subSuite", "value": "TestGetUserOrders"}, {"name": "host", "value": "DESKTOP-42BDC19"}, {"name": "thread", "value": "23912-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_get_user_orders"}]}